cmd/run.go

```diff
@@ -36,16 +38,19 @@ var runCmd = &cobra.Command{
 			wantedOutput = "normal"
 		}
 		if id != "" {
-			log.Fatal().Msgf("--id is deprecated in favour of --include|-i")
+			cmd.SilenceUsage = false
+			return errors.New("--id is deprecated in favour of --include|-i")
 		}
 		if exclude != "" && include != "" {
-			log.Fatal().Msgf("You need to choose one: use --include (%s) or --exclude (%s)", include, exclude)
+			cmd.SilenceUsage = false
+			return fmt.Errorf("You need to choose one: use --include (%s) or --exclude (%s)", include, exclude)
 		}
 		files := fmt.Sprintf("%s/**/*.yaml", dir)
 		tests, err := test.GetTestsFromFiles(files)
 
 		if err != nil {
 			log.Fatal().Err(err)
+			return err
```

Comment:
This will never be reached, will it? `Fatal` calls `Exit`.

Reply:
Of course, propagation is the key here.